AirCasting/Utils/ThresholdSelector.swift

```diff
@@ -0,0 +1,13 @@
+// Created by Lunar on 16/07/2021.
+//
+
+import Foundation
+
+func thresholdFor(thresholds: [SensorThreshold], stream: MeasurementStreamEntity?) -> SensorThreshold? {
```

Comment:
let put it inside a class, a struct or an extension, global funcs should be reserved for a truely global stuff like logging, asserts etc.:

```Swift
extension Array where Element == SensorThreshold {
    func threshold(for stream: MeasurementStreamEntity?) -> SensorThreshold? {
        ...
    }
}
```